_tags

```diff
@@ -9,6 +9,8 @@ not <src/ssl/*>: safe_string
 # to depend on how Ocamlbuild internally handles lists of warn() tags.
 <src/camlp4/*.ml> or <src/ppx/*.ml>: warn(-4)
 <src/camlp4/*.ml>: warn(-3)
+# Warning 4 is disabled due to https://caml.inria.fr/mantis/view.php?id=7451.
+<src/core/lwt.ml>: warn(-4)
```

Comment:
Yes, but I guess it will require messing with `myocamlbuild.ml`, to pass the flag conditionally. Maybe we can just add a comment here in `_tags`, saying that this is only needed as long as Lwt supports OCaml < 4.05 (which might be a long while).

There is also the possibility that the jbuilder port will preempt having to do it with Ocamlbuild, but I don't know yet if this is easy to do with jbuilder.